test/testinput/bufr_ncep_adpupa_nc004005.yaml

```diff
@@ -0,0 +1,277 @@
+          windDirection:
+            query: "NC004005/WDIR"
+          windSpeed:
+            query: "NC004005/WSPD"
+
+          cloudAmount:
+            query: "NC004005/AFCLD/CLAM"
+          cloudType:
+            query: "NC004005/AFCLD/CLTP"
+          heightOfBaseOfCloud:
+            query: "NC004005/AFCLD/HOCB"
+          heightOfTopOfCloud:
+            query: "NC004005/AFCLD/HOCT"
+
+          airframeIcing:
+            query: "NC004005/AFICG/AFIC"
+          heightOfBaseOfIcing:
+            query: "NC004005/AFICG/HBOI"
+          heightOfTopOfIcing:
+            query: "NC004005/AFICG/HTOI"
+
+          moistureQualityMarker:
+            query: "NC004005/QMDD"
+          dewpointTemperature:
+            query: "NC004005/TMDP"
+          relativeHumidity:
+            query: "NC004005/REHU"
+
+          surfaceWindDirection:
+            query: "NC004005/ARSWD/WDIR1"
+          surfaceWindSpeed:
+            query: "NC004005/ARSWD/WSPD1"
+
+          verticalSoundingSignificance:
+            query: "NC004005/VSIG"
+          pressure:
+            query: "NC004005/PRLC"
+
+
+    ioda:
+      backend: netcdf
+      obsdataout: "/scratch1/NCEPDEV/da/Praveen.Kumar/JEDI/IODA_CONVR_ADPUPA_BUFR_2020110112/testoutput/bufr_ncep_adpupa_nc004005.nc"
+
+#      dimensions:
+#        - name: rawrpt_Dim
+#          path: "*/RAWRPT"
+#        - name: afcld_Dim
+#          path: "*/AFCLD"
+
+
+      variables:
+
+#        - name: "MetaData/dateTime"
+#          coordinates: "longitude latitude"
+#          source: variables/timestamp
+#          longName: "Datetime"
+#          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/degreeOfTurbulence"
+          coordinates: "longitude latitude"
+          source: variables/degreeOfTurbulence
+          longName: "Degree Of Turbulence"
+          units: ""
+
+        - name: "MetaData/presentWeather"
+          coordinates: "longitude latitude"
+          source: variables/presentWeather
+          longName: "Present weather"
+          units: ""
+
+        - name: "ObsValue/horizontalVisibility"
+          coordinates: "longitude latitude"
+          source: variables/horizontalVisibility
+          longName: "Horizontal Visibility"
+          units: "m"
+
+        - name: "ObsValue/pressureReducedToMeanSeaLevel"
+          coordinates: "longitude latitude"
+          source: variables/pressureReducedToMeanSeaLevel
+          longName: "Mean Sea-Level Pressure"
+          units: "Pa"
+
+        - name: "QualityMarker/seaSurfaceTemperature"
+          coordinates: "longitude latitude"
+          source: variables/seaSurfaceTemperatureQualityMarker
+          longName: "Sdmedit Quality Mark For Sea Surface Temperature"
+          units: ""
+
+        - name: "ObsValue/seaSurfaceTemperature"
+          coordinates: "longitude latitude"
+          source: variables/seaSurfaceTemperature
+          longName: "Sea/Water Temperature"
+          units: "Kelvin"
```

Comment:
@emilyhcliu @PraveenKumar-NOAA should this be "oceanTemperature" or "seaSurfaceTemperature" ?